test/webpack.config.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
```

Comment:
can this also go in `test/browser`?

Reply:
It could, but it'd prolly fit better in `integration`. I'm okay with leaving it in `test` for now.